roles/youtubedl_material/defaults/main.yml

```diff
@@ -0,0 +1,157 @@
+---
+################################
+# Basics
+################################
+
+youtubedl_material_name: ytdl
+
+################################
+# Paths
+################################
+
+youtubedl_material_paths_folder: "{{ youtubedl_material_name }}"
+youtubedl_material_paths_location: "{{ server_appdata_path }}/{{ youtubedl_material_paths_folder }}"
+youtubedl_material_download_folder: "/mnt/unionfs/Media/{{ youtubedl_material_name }}"
+youtubedl_material_paths_folders_list:
+  - "{{ youtubedl_material_paths_location }}"
+  - "{{ youtubedl_material_paths_location }}/appdata"
+  - "{{ youtubedl_material_paths_location }}/users"
+  - "{{ youtubedl_material_download_folder }}"
+  - "{{ youtubedl_material_download_folder }}/audio"
+  - "{{ youtubedl_material_download_folder }}/video"
+  - "{{ youtubedl_material_download_folder }}/subscriptions"
+
+
+################################
+# Web
+################################
+
+youtubedl_material_web_subdomain: "{{ youtubedl_material_name }}"
+youtubedl_material_web_domain: "{{ user.domain }}"
+youtubedl_material_web_port: "17442"
+youtubedl_material_web_url: "{{ 'https://' + youtubedl_material_web_subdomain + '.' + youtubedl_material_web_domain
+                     if (youtubedl_material_web_subdomain | length > 0)
+                     else youtubedl_material_web_domain }}"
+
+################################
+# DNS
+################################
+
+youtubedl_material_dns_record: "{{ youtubedl_material_web_subdomain }}"
+youtubedl_material_dns_zone: "{{ youtubedl_material_web_domain }}"
+youtubedl_material_dns_proxy: "{{ dns.proxied }}"
+
+################################
+# Traefik
+################################
+
+youtubedl_material_traefik_sso_middleware: "{{ traefik_default_sso_middleware }}"
+youtubedl_material_traefik_middleware: "{{ traefik_default_middleware + ',' + youtubedl_material_traefik_sso_middleware
+                                if (youtubedl_material_traefik_sso_middleware | length > 0)
+                                else traefik_default_middleware }}"
+youtubedl_material_traefik_certresolver: "{{ traefik_default_certresolver }}"
+youtubedl_material_traefik_enabled: true
+
+################################
+# Docker
+################################
+
+# Container
+youtubedl_material_docker_container: "{{ youtubedl_material_name }}"
+
+# Image
+youtubedl_material_docker_image_pull: true
+youtubedl_material_docker_image_tag: "latest"
+youtubedl_material_docker_image: "tzahi12345/youtubedl-material:{{ youtubedl_material_docker_image_tag }}"
+
+# Ports
+youtubedl_material_docker_ports_defaults: []
+youtubedl_material_docker_ports_custom: []
+youtubedl_material_docker_ports: "{{ youtubedl_material_docker_ports_defaults
+                             + youtubedl_material_docker_ports_custom }}"
+
+# Envs
+youtubedl_material_docker_envs_default:
+  PUID: "{{ uid }}"
+  PGID: "{{ gid }}"
+  TZ: "{{ tz }}"
+  # ytdl_use_local_db: 'false'
```

Comment:
This is useless, either implement a variable to toggle this behavior or remove it